README.md

```diff
@@ -39,11 +44,50 @@ discovery in human whole-genome sequencing (WGS) and exome sequencing data.
 - Summary Metrics
 
 ### Software version requirements :
-- GATK 4.beta.5 or later for PairedSingleSampleWF. GATK4.0.1.0 or later for JointGenotypingWf.
-- Picard 2.x
-- Samtools (see gotc docker)
+- GATK 4.beta.5 for PairedSingleSampleWF. GATK4.0.11.0 for JointGenotypingWf.
+- Picard 2.16.0-SNAPSHOT
+- Samtools 1.3.1
 - Python 2.7
+- Cromwell version support 
+  - Successfully tested on v37
+  - Does not work on versions < v23 due to output syntax
+
+### Important Note :
+- VQSR wiring. The SNP and INDEL models are built in parallel, but then the corresponding 
```

Comment:
This is so long, maybe it would be better as a forum article?  Especially since it's more of an FYI.

Reply:
If removing the note doesn't doesn't impede a user from running the workflow then this shouldn't be a problem, we can simply add in a link to the article. This would make the note available to anyone interested in VQSR and not just to those running this particular workflow.